packages/connectivity/connectivity_macos/pubspec.yaml

```diff
@@ -12,7 +11,7 @@ flutter:
 
 environment:
   sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.9.1+hotfix.6 <2.0.0"
+  flutter: "^1.10.0"
```

Comment:
Did you mean to remove the <2.0.0 here?

Reply:
I don't think I can use ^ and <2.0.0 together